activerecord/test/cases/adapters/postgresql/comment_test.rb

```diff
@@ -0,0 +1,87 @@
+require 'cases/helper'
+require 'support/schema_dumping_helper'
+
+class PostgresqlCommentTest < ActiveRecord::PostgreSQLTestCase
```

Comment:
@matthewd is there example of such a generic test with exceptions for some adapters in ActiveRecord?

I have tried to make a single test here: https://github.com/Envek/rails/commit/0db3d7b80216ee164f2e977cfa1b2db5d2064f6b#diff-b905e321f988197140400a200a951ec0

But for some reason these tests doesn't run for MySQL:
```
➜  activerecord  ARCONN=mysql2 bundle exec ruby -Itest test/cases/comment_test.rb
Using mysql2
Run options: --seed 20982

# Running:



Finished in 0.001537s, 0.0000 runs/s, 0.0000 assertions/s.

0 runs, 0 assertions, 0 failures, 0 errors, 0 skips
```
`ActiveRecord::Base.connection.supports_comments?` at https://github.com/Envek/rails/commit/0db3d7b80216ee164f2e977cfa1b2db5d2064f6b#diff-b905e321f988197140400a200a951ec0R29 returns true

PostgreSQL is fine:
```
➜  activerecord  ARCONN=postgresql bundle exec ruby -Itest test/cases/comment_test.rb
Using postgresql
Run options: --seed 21355

# Running:

......

Finished in 1.961782s, 3.0584 runs/s, 11.7240 assertions/s.

6 runs, 23 assertions, 0 failures, 0 errors, 0 skips
```